gravitee-am-test/specs/management/password-policy.jest.spec.ts

```diff
@@ -0,0 +1,170 @@
+import * as faker from 'faker';
+import { afterAll, beforeAll, expect } from '@jest/globals';
+import { requestAdminAccessToken } from '@management-commands/token-management-commands';
+import { createDomain, deleteDomain, startDomain } from '@management-commands/domain-management-commands';
+import { createIdp } from '@management-commands/idp-management-commands';
+import {
+  createPasswordPolicy,
+  deletePasswordPolicy,
+  getAllPasswordPolicies,
+  getAllPasswordPoliciesRaw,
+  getPasswordPolicy,
+  getPasswordPolicyRaw,
+  setPasswordPolicyDefault, updatePasswordPolicy,
+} from '@management-commands/password-policy-management-commands';
+
+global.fetch = fetch;
+
+let accessToken;
+let domain;
+
+beforeAll(async () => {
+  const adminTokenResponse = await requestAdminAccessToken();
+  accessToken = adminTokenResponse.body.access_token;
+  expect(accessToken).toBeDefined();
+
+  const createdDomain = await createDomain(accessToken, 'domain-idp', faker.company.catchPhraseDescriptor());
+  expect(createdDomain).toBeDefined();
+  expect(createdDomain.id).toBeDefined();
+
+  const domainStarted = await startDomain(createdDomain.id, accessToken);
+  expect(domainStarted).toBeDefined();
+  expect(domainStarted.id).toEqual(createdDomain.id);
+
+  domain = domainStarted;
+});
+
+describe('password policy management', () => {
+  it('must return no password policy', async () => {
+    const passwordPolicies = await getAllPasswordPoliciesRaw(domain.id, accessToken);
+    expect(passwordPolicies.raw.status).toEqual(204);
+  });
+
+  it('must create password policy', async () => {
+    const passwordPolicy = await createPasswordPolicy(domain.id, accessToken, {
+      name: 'default',
+    });
+    expect(passwordPolicy).toBeDefined();
+    expect(passwordPolicy.id).toEqual(passwordPolicy.id);
+    expect(passwordPolicy.defaultPolicy).toBeTruthy();
+  });
+
+  it('must update password policy', async () => {
+    const oldPasswordPolicy = await createPasswordPolicy(domain.id, accessToken, {
+      name: 'new',
+    });
+    const updatedPasswordPolicy = await updatePasswordPolicy(domain.id, accessToken, oldPasswordPolicy.id, {
+      name: 'updated-new',
+      defaultPolicy: true,
+    });
+    expect(updatedPasswordPolicy).toBeDefined();
+    expect(updatedPasswordPolicy.id).toEqual(updatedPasswordPolicy.id);
+    expect(updatedPasswordPolicy.defaultPolicy).toBeTruthy();
+  });
+
+  it('must find all password policies', async () => {
+    const idpSet = await getAllPasswordPolicies(domain.id, accessToken);
+    expect(idpSet.length).toBeGreaterThan(0);
+  });
+
+  it('must delete password policy', async () => {
+    const passwordPolicy = await createPasswordPolicy(domain.id, accessToken, {
+      name: 'policy-to-delete',
+    });
+    await deletePasswordPolicy(domain.id, accessToken, passwordPolicy.id);
+    await expect(async () => {
+      await getPasswordPolicyRaw(domain.id, accessToken, passwordPolicy.id);
+    }).rejects.toThrow();
```

Comment:
**question**: is there any wait to check the status to be sure we are receiving 404.

Reply:
yes, fixed